stdlib/2/collections.pyi

```diff
@@ -93,10 +93,13 @@ class Counter(Dict[_T, int], Generic[_T]):
     def __iand__(self, other: Counter[_T]) -> Counter[_T]: ...
     def __ior__(self, other: Counter[_T]) -> Counter[_T]: ...
 
+_OrderedDictT = TypeVar('_OrderedDictT', bound=OrderedDict)
+
 class OrderedDict(Dict[_KT, _VT], Reversible[_KT], Generic[_KT, _VT]):
     def popitem(self, last: bool = ...) -> Tuple[_KT, _VT]: ...
     def __reversed__(self) -> Iterator[_KT]: ...
-    def __copy__(self) -> OrderedDict[_KT, _VT]: ...
+    def __copy__(self: _OrderedDictT) -> _OrderedDictT: ...
```

Comment:
There is actually no `Copyable` protocol in typing, so I would say it is safe to remove it, if it is not defined at runtime.

Reply:
will do.